linera-service/tests/end_to_end_tests.rs

```diff
@@ -1215,32 +1238,31 @@ async fn test_end_to_end_fungible() {
     assert_eq!(chain2, client2.assign(client2key, effect_id).await.unwrap());
 
     // The players
-    let owner1 = client1.get_owner().expect("Failed to get the owner");
+    let owner1 = client1.get_owner().unwrap();
```

Comment:
We're using `unwrap` pretty much everywhere in tests and that's fine.

Reply:
Ok.